From f9abff6a3b37f632c0c1ec213d1c808a2340e884 Mon Sep 17 00:00:00 2001
From: Bart Ribbers <bribbers@disroot.org>
Date: Fri, 10 Oct 2025 23:42:40 +0200
Subject: [PATCH] apk: remove non-existing ::hasCredentials override property

It has been removed in 8519e5ccfa624f029d81e13808968ed8b3354135
but seems to have been forgotten to remove in the apk backend.
---
 .../backends/AlpineApkBackend/AlpineApkReviewsBackend.h       | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/libdiscover/backends/AlpineApkBackend/AlpineApkReviewsBackend.h b/libdiscover/backends/AlpineApkBackend/AlpineApkReviewsBackend.h
index 6c7c221ad..520883e46 100644
--- a/libdiscover/backends/AlpineApkBackend/AlpineApkReviewsBackend.h
+++ b/libdiscover/backends/AlpineApkBackend/AlpineApkReviewsBackend.h
@@ -29,10 +29,6 @@ public:
     }
 
     Q_SCRIPTABLE Rating ratingForApplication(AbstractResource *) const override;
-    bool hasCredentials() const override
-    {
-        return false;
-    }
     void deleteReview(Review *) override
     {
     }
-- 
2.51.0

